agent/handlers/gql/schema.go

```diff
@@ -397,12 +486,47 @@ func taskContainersResolver(state dockerstate.TaskEngineState) func(p graphql.Re
 			for _, dockerContainer := range containerNameToDockerContainer {
 				resp = append(resp, dockerContainer)
 			}
+			pulledContainers, _ := state.PulledContainerMapByArn(task.Arn)
+			for _, dockerContainer := range pulledContainers {
+				resp = append(resp, dockerContainer)
+			}
 			return resp, nil
 		}
 		return nil, nil
 	}
 }
 
+func taskTagsResolver(ecsClient api.ECSClient) func(p graphql.ResolveParams) (interface{}, error) {
+	return func(p graphql.ResolveParams) (interface{}, error) {
+		if task, ok := p.Source.(*apitask.Task); ok {
+			taskTags, err := ecsClient.GetResourceTags(task.Arn)
+			if err == nil {
```

Comment:
nit: let's do `if err != nil`, like we do in most cases. Same for `containerInstanceTagsResolver`